### ISSUE TITLE
Add more eval metrics

### ISSUE THREAD

**Issue by pplonski** [mljar/mljar-supervised#268]
Please add support for more eval metrics.

**Comment by girardea** [comment]
Hi @pplonski, first of all: mljar-supervised is a fantastic and much useful tool! Thanks for the good work!

I have a case where I would like to use [Mean Absolute Percentage Error](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.mean_absolute_percentage_error.html#sklearn.metrics.mean_absolute_percentage_error). It is currently not present and I tried tweaking things in metric.py in order to make it available but it triggered errors in callbacks that I was not sure to fully understand.

Anyway, it'd be great to have this metric available, and I'd be glad to help!

**Comment by pplonski** [comment]
Hi @girardea! I've added an issue for MAPE https://github.com/mljar/mljar-supervised/issues/310 - I will add it very soon.

**Comment by girardea** [comment]
That's great new! Thanks!

**Comment by girardea** [comment]
Tested and approved 😉

**Comment by pplonski** [comment]
@girardea great! The changes will go to the `0.9.0` version. Thank you for your help.

**Comment by pplonski** [comment]
More metrics added in `0.10.3`.
